app/db/database.sql

```diff
@@ -1,4 +1,303 @@
-CREATE TABLE IF NOT EXISTS [Patient] ( 
-	[id] INTEGER  NOT NULL PRIMARY KEY AUTOINCREMENT, 
-	[name] NVARCHAR(50)  NOT NULL
-);
\ No newline at end of file
+/* Table creation */
+CREATE TABLE IF NOT EXISTS Doctor
+(
+  id                  INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  physician_permit_nb INTEGER NOT NULL,
+  first_name          TEXT    NOT NULL,
+  last_name           TEXT    NOT NULL,
+  specialty           TEXT    NOT NULL,
+  city                TEXT    NOT NULL,
+  password TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS Patient
+(
+  id             INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  health_card_nb INTEGER NOT NULL,
+  date_of_birth  TEXT    NOT NULL,
+  gender         TEXT    NOT NULL,
+  phone_nb       TEXT    NOT NULL,
+  home_address   TEXT    NOT NULL,
+  email          TEXT    NOT NULL,
+  first_name     TEXT    NOT NULL,
+  last_name      TEXT    NOT NULL,
+  password TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS Availability
+(
+  id        INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  doctor_id INTEGER NOT NULL,
+  start     TEXT    NOT NULL,
+  room      TEXT    NOT NULL,
+  free      INTEGER NOT NULL,
+  date      TEXT    NOT NULL,
```

Comment:
so we're going with a date string? it would be easier for filtering if we had this split into month, day, year, otherwise we will have to pull all the DB data and query in RAM, which has a large overhead